internal/common/shared/predictions.go

```diff
@@ -0,0 +1,24 @@
+package shared
+
+// Prediction is a struct containing the necessary parameters for an island to
+// make a prediction about a disaster
+type Prediction struct {
+	CoordinateX float64
+	CoordinateY float64
+	Magnitude   float64
+	TimeLeft    int
+	Confidence  float64
```

Comment:
is `Confidence` bounded? Similar for other values like `Magnitude`.

Reply:
Will approof after the metrics here are defined :)

Reply:
Confidence is bounded between 0 and 100 at the moment, magnitude might be bounded by Team 5

Reply:
coolioo -- can you add a note there? we should actually share the types when these are merged in--can you add a TODO here? Or, just define the type now and get Team 5 to follow it.

(`type Coordinte = int`) in package `shared`

Reply:
Done